File: Lenguaje de Programación 2/Parcial/Semana 3/BinarioC#/Persona.cs
using System;
[Serializable]
public class Persona{
	private string nombre;
	private int edad;
	
	public string Nombre{
		set{
			nombre = value;
		}
		get{
			return nombre;
		}
	}
	
	public int Edad{
		set{
			edad = value;
		}
		get{
			return edad;
		}
	}
}

File: Lenguaje de Programación 2/Parcial/Semana 2/Librerías/Persona/persona.java
package Persona;
public class persona{
	private String nombre;
	private int edad;
	
	public persona(String nombreRecibe, int edadRecibe){
		this.nombre = nombreRecibe;
		this.edad = edadRecibe;
	}
	
	public persona(String nombreRecibe){
		this.nombre = nombreRecibe;
	}
	
	public persona(){
	}
	
	public String getNombre(){
		return nombre;
	}
	
	public void setNombre(String nombreRecibe){
		this.nombre = nombreRecibe;
	}
	
	public int getEdad(){
		return edad;
	}
	
	public void setEdad(int edadRecibe){
		this.edad = edadRecibe;
	}
	
	public void mostrarNombre(){
		System.out.println(this.nombre + " " + this.edad);
	}
}